### ISSUE TITLE
[BUG] Failed to fetch

### ISSUE THREAD

**Issue by dustinvo17** [microsoft/playwright#6297]
**Context:**
- Playwright Version: 1.9
- Operating System: Window
- Node.js version: 10.12
- Browser: Chromium

Hi everyone, I encountered a bug with fetch inside page.evaluate. Initially, I thought it was blocked by the browser, I then set the bypassCSP and ignoreHTTPS to true but the issue still remained. I wonder if there is any fix for this? 

```javascript
    private async seedBackups() {
        await page.evaluate(async (apiURL) => {
            const file = new File(["backup"], "userBackup.json", {
                type: "application/json",
            });

            const formData = new FormData();
            formData.append("userBackup", file);

            const createBackupResponse = await fetch(`${apiURL}/api/v1/backups`, {
                method: "POST",
                body: formData,
            });

            return await createBackupResponse.json();
        }, this.apiURL);
    }
```

**Describe the bug**

page.evaluate: Evaluation failed: TypeError: Failed to fetchError:

**Comment by dgozman** [comment]
Are there any more details? The error message seems to be truncated - mind posting a full one? Also, it would be very nice to have a full repro script, so that we can repro and fix this issue.

**Comment by dustinvo17** [comment]
Hi @dgozman, thank you very much for your response. It turned out the bug is not because of the playwright itself but the OS on my Azure pipeline. I would mark this as closed
